packages/atomic/src/components/search/generated-answer/atomic-generated-answer-feedback/atomic-generated-answer-feedback-modal.pcss

```diff
@@ -0,0 +1,26 @@
+@import '../../../../global/global.pcss';
+
+[part='reason'] {
+  @apply text-base leading-4 text-neutral-dark;
+
+  @screen desktop-only {
+    @apply mt-6;
+  }
+
+  @screen mobile-only {
+    @apply mt-4;
```

Comment:
Things like this can also be applied in the markup directly using `mobile-only:mt-4` FYI